isis/src/mro/apps/hical/hical.cpp

```diff
@@ -217,11 +199,31 @@ namespace Isis {
                            "in your configuration file.";
             throw IException(IException::User, mess, _FILEINFO_);
           }
-          ZeroDarkRate zdr(hiconf);
-          calVars->add(hiconf.getProfileName(), zdr.ref());
-          ZdrHist = zdr.History();
-          if ( hiprof.exists("DumpModuleFile") ) {
-            zdr.Dump(hiconf.getMatrixSource("DumpModuleFile",hiprof));
+          try{
+            ZeroDarkRate zdr(hiconf);
+            calVars->add(hiconf.getProfileName(), zdr.ref());
+            ZdrHist = zdr.History();
+            if ( hiprof.exists("DumpModuleFile") ) {
+              zdr.Dump(hiconf.getMatrixSource("DumpModuleFile",hiprof));
+            }
+          }
+          catch(IException e){
+            if (hiprof.exists("Fallback") && IsTrueValue(hiprof, "Fallback")){
+                zdrFallback = true;
+                calVars->add(hiconf.getProfileName(), HiVector(nsamps, 0.0));
+                std::cerr << "Falling back to ZeroDark implementation. Unable to initialize ZeroDarkRate "
+                          << "module with the following error:" << std::endl << e.what() << "\nContinuing..."<< std::endl;
```

Comment:
Typically we would throw a "Warning" in the logging pvl output, not sure if that is something we want to do here

Reply:
I think as is right now is good. If we want to throw it in the actual app logs later on I don't think that will be very hard